inc/Engine/Preload/Subscriber.php

```diff
@@ -449,6 +449,10 @@ public function unlock_url( string $url ) {
 	public function add_preload_excluded_uri( $regexes ): array {
 		$preload_excluded_uri = (array) $this->options->get( 'preload_excluded_uri', [] );
 
+		if ( ! empty( $_SERVER['REQUEST_URI'] ) && (bool) ! $this->options->get( 'manual_preload', true ) ) {
+			$regexes[] = filter_var( wp_unslash( $_SERVER['REQUEST_URI'] ), FILTER_SANITIZE_URL );
```

Comment:
Why are we adding the URL to the regexes when the option is disabled?

When the option is disabled, we should be doing nothing

Reply:
That's true, I have changed it.